src/SFA.DAS.AANHub.Application/StagedApprentices/Queries/GetStagedApprenticeResult.cs

```diff
@@ -0,0 +1,36 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.StagedApprentices.Queries
+{
+    public class GetStagedApprenticeResult
+    {
+        public long Uln { get; set; }
+        public long ApprenticeshipId { get; set; }
```

Comment:
Missing fields like first name, last name etc?

Reply:
Closing this based on @alandburns response on comms